test/chainsaw/testmetrics/metrics-hpa/chainsaw-test.yaml

```diff
@@ -0,0 +1,30 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/kyverno/chainsaw/main/.schemas/json/test-chainsaw-v1alpha1.json
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  creationTimestamp: null
+  name: metrics-hpa
+spec:
+  steps:
+  - name: step-00
+    try:
+    - apply:
+        file: 00-install.yaml
+    - script:
+        content: |
+          kubectl apply -f mock-server.yaml -n $NAMESPACE
+    - script:
+        content: |
+          echo "registering the metric"
+          envsubst < metric.yaml | kubectl apply -f - -n $NAMESPACE
+          echo "configuring HPA"
+          kubectl apply -f hpa.yaml -n $NAMESPACE
+    catch:
+    - podLogs:
+        selector: app=podtato-head
+    - script:
+        content: |
+          kubectl logs -l app=metrics-operator -n keptn-system
+          kubectl describe keptnmetric podtatometric -n $NAMESPACE
+          kubectl describe keptnmetricsprovider dynatrace -n $NAMESPACE
+          kubectl describe pods -l app=podtato-head -n $NAMESPACE
```

Comment:
I would say this line is a repetition of line 24

Reply:
Yeah, i converted the original kuttl test... including inconsistencies ! 😂